checks.d/statsd.py

```diff
@@ -0,0 +1,78 @@
+# stdlib
+import re
+import socket
+import sys
+from StringIO import StringIO
+
+# project
+from checks import AgentCheck
+
+SERVICE_CHECK_NAME = "statsd.can_connect"
+SERVICE_CHECK_NAME_HEALTH = "statsd.is_up"
+
+ENDER = re.compile("^(END|health: up|health: down)\n$", re.MULTILINE)
+BAD_ENDER = re.compile("^ERROR\n$", re.MULTILINE)
+
+class StatsCheck(AgentCheck):
+    def check(self, instance):
+        host = instance.get("host", "localhost")
+        port = instance.get("port", 8126)
+        tags = instance.get('tags', [])
+
+        # Is it up?
+        health = self._send_command(host, port, "health").getvalue().strip()
+        if health.strip() == "health: up":
+            self.service_check(
+                SERVICE_CHECK_NAME_HEALTH, AgentCheck.OK, []
+            )
+        else:
+            self.service_check(
+                SERVICE_CHECK_NAME_HEALTH, AgentCheck.CRITICAL, []
```

Comment:
I've got `_send_command` doing a `raise` if there are any failures, so the whole thing will error out. Do I need to do more / am I misunderstanding? :)

iow, there's nothing to check because `_send_command` shouldn't return (without an exception) unless it worked.